projects/cdk/utils/miscellaneous/is-empty.ts

```diff
@@ -0,0 +1,9 @@
+import {tuiIsString} from './is-string';
+
+type EmptyValue = '' | null | undefined;
+
+export function tuiCheckValueIsEmpty<T>(value: EmptyValue | T): value is EmptyValue {
```

Comment:
Maybe keep it local function inside `tuiCleanObject`? I'm not sure we want it exported, because it's debatable what to consider "empty" value.

Reply:
fixed